components/WeightComponent/RealWeight.cpp

```diff
@@ -1,28 +1,33 @@
-#include "RealWeight.hpp"
+#include "include/RealWeight.hpp"
 
 using namespace Weight;
 
 RealWeight::RealWeight()
 {
-    hx711_t dev = {
-        .dout = GPIO_NUM_19,
-        .pd_sck = GPIO_NUM_18,
-        .gain = HX711_GAIN_A_128
+    hx711_t devs[4] = {
+        { .dout = GPIO_NUM_19, .pd_sck = GPIO_NUM_18, .gain = HX711_GAIN_A_128 },
+        { .dout = GPIO_NUM_21, .pd_sck = GPIO_NUM_22, .gain = HX711_GAIN_A_128 },
+        { .dout = GPIO_NUM_23, .pd_sck = GPIO_NUM_25, .gain = HX711_GAIN_A_128 },
+        { .dout = GPIO_NUM_26, .pd_sck = GPIO_NUM_27, .gain = HX711_GAIN_A_128 }
```

Comment:
im not sure if this is the current commit but, from what i understand (and correct me if im wrong) this was added in here because there are four load cell sensors? if that's the case, i read through the datasheet and i believe that while there are four load cell sensors, those are connected to the hx711 module which is connected to the esp32 using only two pins (pd_sck, dout). therefore, you only need to initialize one hx711_t struct.